docker-compose.yaml

```diff
@@ -74,11 +74,12 @@ services:
       /bin/sh -c "
       /usr/bin/mc config host add myminio http://minio:9000 minio miniotest;
       /usr/bin/mc mb myminio/eppdata;
+      /usr/bin/mc anonymous set public myminio/eppdata;
```

Comment:
Is this how we intend to do it in production, as opposed to using presigned URLs or a proxy?

Feels a bit blunt and open, but if we're not worried about that then I suppose it's OK. I just assumed we would want some control/audit over the requests made to the bucket.

Reply:
Have removed now. Since: https://github.com/elifesciences/enhanced-preprints-client/pull/584